python/rmm/rmm/_cuda/gpu.py

```diff
@@ -1,14 +1,15 @@
-# Copyright (c) 2020, NVIDIA CORPORATION.
+# Copyright (c) 2020-2024, NVIDIA CORPORATION.
 
-from cuda import cuda, cudart
+import cuda.bindings.driver as driver
+import cuda.bindings.runtime as runtime
```

Comment:
Maybe?
```suggestion
from cuda.bindings import driver, runtime
```

Reply:
Oh, good catch. I was playing with different names earlier. This is the right call!